addons/mail/static/src/components/message/message.xml

```diff
@@ -151,18 +151,7 @@
                                 <ul class="o_Message_trackingValues mt-2">
                                     <t t-foreach="messageView.message.trackingValues" t-as="value" t-key="value.id">
                                         <li>
```

Comment:
Can you remove the `<li>` here?
We have a task to make everything below `t-foreach` encapsulated in a component.
This is desirable to move as much code to models as possible.

Reply:
If you can't and it's due to important styling that needs a container like `<li>`, then could move it to template of `MessageTrackingValue`.

Reply:
Tracking value in another context won't be inside a list though?
If really needed the solution to make an intermediary component for this loop item.

Reply:
Indeed, then the solution is to make a `MessageTrackinValueList` something like this but I will prefer to do it later in another pr

Reply:
> Indeed, then the solution is to make a `MessageTrackinValueList` something like this but I will prefer to do it later in another pr

`li` means "List Item" BTW.

Check whether you really need this `<li>`. If you don't, it's good to just delete it.

Reply:
Yes I know it is an item. It is needed in this case, for example. A state changes and the currency change at the same time. Then there will be two tracking value changes that need to be displayed.

Reply:
Ok, this is handled by the multiplicity of `Message/trackingValues`. The `<li>` is really about style.

Let's keep it that way for now ;)